src/astro/utils/__init__.py

```diff
@@ -1,10 +1,19 @@
+from typing import Type, Union
+
 from airflow.hooks.base import BaseHook
 from airflow.providers.sqlite.hooks.sqlite import SqliteHook
 
-from astro.utils.dependencies import BigQueryHook, PostgresHook, SnowflakeHook
+from astro.utils.dependencies import (
+    BigQueryHook,
+    MissingPackage,
+    PostgresHook,
+    SnowflakeHook,
+)
 
 
-def get_hook(conn_id, database, role=None, schema=None, warehouse=None):
+def get_hook(
+    conn_id, database, role=None, schema=None, warehouse=None
```

Comment:
Should we add typehints to params too, for example as follows?

```
    conn_id: str, database:  Database
```

Reply:
Absolutely, I didn't add further type hints to this module because that's Part 7
(`src/astro/utils/__init__.py` is out of the scope of the current PR)
https://github.com/astro-projects/astro/issues/281

I just added the minimum for the part I was working to work :)

Reply:
Ohh yes :)